test/training_service.yml

```diff
@@ -4,7 +4,7 @@ local:
 remote:
   trainingServicePlatform: remote
   machineList:
-    - ip:
+    - ip: 23.96.57.139
```

Comment:
This value should be written  at run time.

Reply:
removed.